patches/api/0403-Add-entity-knockback-API.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: MelnCat <melncatuwu@gmail.com>
+Date: Sun, 16 Oct 2022 12:10:00 -0700
+Subject: [PATCH] Add entity knockback API
+
+
+diff --git a/src/main/java/org/bukkit/entity/LivingEntity.java b/src/main/java/org/bukkit/entity/LivingEntity.java
+index 319e4571aca24d1e3f6c85b7435d65c0e77a5245..296dbc0360f380820ec5f2b569353251a11f8547 100644
+--- a/src/main/java/org/bukkit/entity/LivingEntity.java
++++ b/src/main/java/org/bukkit/entity/LivingEntity.java
+@@ -1003,5 +1003,16 @@ public interface LivingEntity extends Attributable, Damageable, ProjectileSource
+             this.swingOffHand();
+         }
+     }
++
++    /**
++     * Knocks back this entity from a specific direction with a specified strength.
++     *
++     * The direction specified in this method will be the direction of the source of the knockback,
++     * so the entity will be pushed in the opposite direction.
++     * @param strength The strength of the knockback. If the entity hsa knockback resistance, this will be reduced.
```

Comment:
A little typo: "If the entity **hsa** knockback resistance, this will be reduced"

Reply:
Fixed